### PR TITLE
Platform bulk load updates

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Mooring additions for Pioneer 18, Irminger 9 and Papa 9 cruises.
Note: PMCI-16 and PMUI-17 were already added previously.  Also, Pioneer 19 is recovery-only so there was just one set of moorings to add for Pioneer 2022 cruises.

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
[Pioneer 18 AM Updates [DT-152]](https://trello.com/c/zFi7BfCv/152-pioneer-18-am-updates-dt-152)

**Comment by rtravis-whoi** [comment]
[Papa 9 AM Updates [DT-161]](https://trello.com/c/n2lTwbTF/161-papa-9-am-updates-dt-161)

**Comment by rtravis-whoi** [comment]
[Irminger 9 AM Updates [DT-160]](https://trello.com/c/YNyU7qml/160-irminger-9-am-updates-dt-160)

File: bulk/platform_bulk_load-AssetRecord.csv [hunk 1]
```diff
@@ -317,6 +322,7 @@ CGMCP-04OSSM-00011,,Platform,0,Coastal Pioneer Offshore Surface Mooring,MOORING
 CGMCP-04OSSM-00012,,Platform,0,Coastal Pioneer Offshore Surface Mooring,MOORING PIONEER OS,WHOI,CP04OSSM,CP04OSSM-00012,,,,,Woods Hole Oceanographic Institution
 CGMCP-04OSSM-00013,,Platform,0,Coastal Pioneer Offshore Surface Mooring,MOORING PIONEER OS,WHOI,CP04OSSM,CP04OSSM-00013,,,,,Woods Hole Oceanographic Institution
 CGMCP-04OSSM-00014,,Platform,0,Coastal Pioneer Offshore Surface Mooring,MOORING PIONEER OS,WHOI,CP04OSSM,CP04OSSM-00014,,,,,Woods Hole Oceanographic Institution
+CGMCP-04OSSM-00015,,Platform,0,Coastal Pioneer Offshore Surface Mooring,MOORING PIONEER OS,WHOI,CP04OSSM,CP04OSSM-00015,,,,,Woods Hole Oceanographic Institution
```
Comment (snwhite) [inline bulk/platform_bulk_load-AssetRecord.csv:325]: All Pioneer moorings appear correct.

File: bulk/platform_bulk_load-AssetRecord.csv [hunk 2]
```diff
@@ -361,6 +370,7 @@ CGMGI-03FLMB-00005,,Platform,0,Global Irminger Sea Flanking Subsurface Mooring B
 CGMGI-03FLMB-00006,,Platform,0,Global Irminger Sea Flanking Subsurface Mooring B,MOORING MESCALE FLANKING B SP,WHOI,GI03FLMB,GI03FLMB-00006,,,,,Woods Hole Oceanographic Institution
 CGMGI-03FLMB-00007,,Platform,0,Global Irminger Sea Flanking Subsurface Mooring B,MOORING MESCALE FLANKING B SP,WHOI,GI03FLMB,GI03FLMB-00007,,,,,Woods Hole Oceanographic Institution
 CGMGI-03FLMB-00008,,Platform,0,Global Irminger Sea Flanking Subsurface Mooring B,MOORING MESCALE FLANKING B SP,WHOI,GI03FLMB,GI03FLMB-00008,,,,,Woods Hole Oceanographic Institution
+CGMGI-03FLMB-00009,,Platform,0,Global Irminger Sea Flanking Subsurface Mooring B,MOORING MESCALE FLANKING B SP,WHOI,GI03FLMB,GI03FLMB-00009,,,,,Woods Hole Oceanographic Institution
```
Comment (snwhite) [inline bulk/platform_bulk_load-AssetRecord.csv:373]: All Irminger moorings appear correct.

File: bulk/platform_bulk_load-AssetRecord.csv [hunk 3]
```diff
@@ -385,6 +397,7 @@ CGMGP-03FLMB-00005,,Platform,0,Global Station Papa Flanking Subsurface Mooring B
 CGMGP-03FLMB-00006,,Platform,0,Global Station Papa Flanking Subsurface Mooring B,MOORING MESCALE FLANKING B SP,WHOI,GP03FLMB,GP03FLMB-00006,,,,,Woods Hole Oceanographic Institution
 CGMGP-03FLMB-00007,,Platform,0,Global Station Papa Flanking Subsurface Mooring B,MOORING MESCALE FLANKING B SP,WHOI,GP03FLMB,GP03FLMB-00007,,,,,Woods Hole Oceanographic Institution
 CGMGP-03FLMB-00008,,Platform,0,Global Station Papa Flanking Subsurface Mooring B,MOORING MESCALE FLANKING B SP,WHOI,GP03FLMB,GP03FLMB-00008,,,,,Woods Hole Oceanographic Institution
+CGMGP-03FLMB-00009,,Platform,0,Global Station Papa Flanking Subsurface Mooring B,MOORING MESCALE FLANKING B SP,WHOI,GP03FLMB,GP03FLMB-00009,,,,,Woods Hole Oceanographic Institution
```
Comment (snwhite) [inline bulk/platform_bulk_load-AssetRecord.csv:400]: All Papa moorings appear correct.

**Review by snwhite — APPROVED** [review]

File: bulk/platform_bulk_load-AssetRecord.csv (hunk 1 above)
Comment (cdobs) [inline bulk/platform_bulk_load-AssetRecord.csv:325]: Pioneer looks good

File: bulk/platform_bulk_load-AssetRecord.csv (hunk 2 above)
Comment (cdobs) [inline bulk/platform_bulk_load-AssetRecord.csv:373]: Irminger looks good

File: bulk/platform_bulk_load-AssetRecord.csv (hunk 3 above)
Comment (cdobs) [inline bulk/platform_bulk_load-AssetRecord.csv:400]: Papa looks good

**Review by cdobs — APPROVED** [review]

**Pull request merged**